CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Tray: add missing Swift variables in the secret
 - Tray: fix pods anti-affinity selector
+- Remove pandas dependancy
```

Comment:
May be we could use the `removed` section for this ? WDYT?
```suggestion

### Removed
- Remove pandas dependency
```

Reply:
Indeed, that would be a good idea. As I saw in previous modifications that remove was in change, I did the same but you're right. It's better ;)

Reply:
That was a mistake then. Can you fix it?